src/rvsdg/egglog_optimizer/constant_fold.rs

```diff
@@ -0,0 +1,20 @@
+use super::BRIL_OPS;
+
+pub(crate) fn constant_fold_egglog() -> String {
+    let mut res = vec![];
+
+    for bril_op in BRIL_OPS {
+        let op = bril_op.op;
+        let egglog_op = bril_op.egglog_op;
+        if bril_op.num_inputs() == 2 {
+            res.push(format!(
+                "(rewrite ({op} ty
```

Comment:
Alas if only we had metaprogramming in egglog.

Reply:
alas